### PR TITLE
Add option to search for users is active join a team

### PR DESCRIPTION
**Pull request by tyroneyeh**
Adding a user in a team to enter a username gives a list of no active users

### CODE REVIEW

**Comment by wolfogre** [comment]
If I haven't missed anything, the `/user/search` is used by `#search-user-box`, the box is used by `team/members.tmpl` and `collaboration.tmpl`.

It does make sense to search for active users only when adding members or collaborators, but I still worry that it could be used for other purposes and confuse the developer in the future. How about adding a new parameter to the url to indicate whether active user only?

**Comment by tyroneyeh** [comment]
> If I haven't missed anything, the `/user/search` is used by `#search-user-box`, and the box is used by `team/members.tmpl` and `collaboration.tmpl`.
> 
> It does make sense to search for active users only when adding members or collaborators, but I still worry that it could be used for other purposes and confuse the developers in the future. How about adding a new parameter to the url to indicate whether active user only?

Thanks, I have edited it again, is that right?

File: web_src/js/features/comp/SearchUserBox.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export function initCompSearchUserBox() {
   $searchUserBox.search({
     minCharacters: 2,
     apiSettings: {
-      url: `${appSubUrl}/user/search?q={query}`,
+      url: `${appSubUrl}/user/search?q={query}&a=1`,
```
Comment (wolfogre) [inline web_src/js/features/comp/SearchUserBox.js]: Maybe:
```suggestion
      url: `${appSubUrl}/user/search?active=1&q={query}`,
```

File: routers/web/user/search.go [hunk 2]
```diff
@@ -24,6 +24,7 @@ func Search(ctx *context.Context) {
 		Keyword:     ctx.FormTrim("q"),
 		UID:         ctx.FormInt64("uid"),
 		Type:        user_model.UserTypeIndividual,
+		IsActive:    ctx.FormOptionalBool("a"),
```
Comment (wolfogre) [inline routers/web/user/search.go]: And:
```suggestion
		IsActive:    ctx.FormOptionalBool("active"),
```

**Review by wolfogre — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
@wolfogre this isn't breaking as it is the UI api, which external applications shouldn't be using.

**Comment by techknowlogick** [comment]
🤖 🎺

**Pull request merged**